docs/tutorials/vagrant.rst

```diff
@@ -0,0 +1,740 @@
+      "Version": "To be filled by O.E.M.",
+      "Serial Number": "SerialNumber",
+      "UUID": "75277866-7C0D-1000-A5B1-2C600C8374BD",
+      "Wake-up Type": "Power Switch",
+      "SKU Number": "To be filled by O.E.M.",
+      "Family": "To be filled by O.E.M."
+    }
+
+Adding a SKU definition
+-------------------------
+
+All this geneaology can be used to create rules that will uniquely identify a
+type of machine, a feature which we call **SKU** in RackHD. When a node is discovered,
+one the last steps of the built in discovery workflow is to compare the node against
+all existing SKU definitions. If the SKU definition maps, it is applied. Only one
+SKU will be assigned to a node at a time - there's a similiar feature called **tag**
+that can be used to group multiple nodes with the same characteristics.
+
+You can get more details on SKUs at :doc:`../rackhd2.0/skus`, and tags at :doc:`../rackhd2.0/tags`.
+
+One of the specific benefits of a SKU is that you can define an additional workflow
+to be invoked as soon as the node is discovered, providing RackHD with an explicit
+set of tasks to follow when the node is identified. This is how you set up RackHD to
+automatically CentOS, for example.
+
+You create a SKU by uploading a specially structured bit of JSON through the API. In
+the `example/` directory, we have a few specific examples pre-set to work with this
+tutorial. For this example, we'll add in a specific workflow to install CentOS, and a
+SKU definition which will use the simulated hardware to trigger that workflow.
+
+.. sidebar:: Installing the CentOS Installation ISO
+
+    To operate correctly, the CentOS install workflow we just added expects to find the
+    OS installation files in the directory ``/CentOS/7.0``, which doesn't exist by default
+    on our instance of RackHD.
+
+    We can install the relevant files by downloading and unpacking a CentOS installation
+    ISO. To do so, log into your instance of RackHD using a command like ``vagrant dev ssh``
+    and then invoke the following commands::
+
+        sudo mkdir -p /var/mirrors
+        cd /tmp
+        wget http://mirrors.mit.edu/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-1511.iso
+        # 4GB download!
+        sudo python ~/src/on-tools/scripts/setup_iso.py \
+        /tmp/CentOS-7-x86_64*.iso /var/mirrors \
+        --link=/home/vagrant/src
+
+- add the workflow to install CentOS for this specific SKU::
+
+    cd ~/src/rackhd/example
+    # make sure you're in the example directory to reference the sample JSON correctly
+
+    curl -H "Content-Type: application/json" \
+    -X PUT --data @samples/vQuanta_install_centos.json \
+    http://localhost:9090/api/1.1/workflows
+
+The `vQuanta_install_centos.json` sample
+
+.. code-block:: JSON
+
+    {
+        "friendlyName": "vQuanta D51 Default Install CentOS",
+        "injectableName": "Graph.DefaultVQuanta.InstallCentOS",
+        "options": {
+          "defaults": {
```

Comment:
I suggest putting all options into the label `intall-centos`, so that the fresh user will not confuse why some options lie in `default`, some in `install-centos`.
Use `install-centos` rather than `defaults` because it is more straightforward these are options for CentOS installation.